src/test/kotlin/lotto/domain/LottoStoreTest.kt

```diff
@@ -0,0 +1,15 @@
+package lotto.domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class LottoStoreTest {
+
+    @Test
+    fun `로또를 판매합니다`() {
+        val store = LottoStore { Lotto.with(listOf(1, 2, 3, 4, 5, 6)) }
+
+        assertThat(store.buy(14000).lottos).containsOnly(Lotto.with(listOf(1, 2, 3, 4, 5, 6)))
+        assertThat(store.buy(14000).lottos.size).isEqualTo(14)
```

Comment:
```suggestion
        val store = LottoStore { Lotto.with(listOf(1, 2, 3, 4, 5, 6)) }
        val lottos = store.buy(14000).lottos
        assertThat(lottos).containsOnly(Lotto.with(listOf(1, 2, 3, 4, 5, 6)))
        assertThat(lottos.size).isEqualTo(14)
```

상점에서 구매한 로또를 이용하여 테스트를 하면 괜찮지 않을까요?